requirements.txt

```diff
@@ -0,0 +1,22 @@
+astroid==2.4.2
```

Comment:
We should be using poetry rather than pip on this project (apologies if the project exercise brief is out of date).

E.g. add `requests` to your `pyproject.toml` by running `poetry add requests`.

Reply:
This is now fixed and new pull request will be raised

Reply:
Changes checked in. Won't raise a new pull request as I don't think it will be needed